api_app/script_analyzers/observable_analyzers/dnstwist.py

```diff
@@ -0,0 +1,58 @@
+import subprocess
+import json
+import logging
+from shutil import which
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+
+
+class DNStwist(classes.ObservableAnalyzer):
+    command: str = "dnstwist"
+    dictionary_base_path: str = "/opt/deploy/dnstwist-dictionaries/"
+    final_report = {}
+
+    def set_config(self, additional_config_params):
+        self._ssdeep = additional_config_params.get("ssdeep", False)
+        self._mxcheck = additional_config_params.get("mxcheck", False)
+        self._tld = additional_config_params.get("tld", False)
+        self._tld_dict = additional_config_params.get("tld_dict", "abused_tlds.dict")
+
+    def run(self):
+        if not which(self.command):
+            self.report["success"] = False
+            raise AnalyzerRunException("dnstwist not installed!")
+
+        args = [self.command, "--registered", "--format", "json"]
+        if self._ssdeep:
+            args.append("--ssdeep")
+            self.final_report["ssdeep"] = True
+        if self._mxcheck:
+            args.append("--mxcheck")
+            self.final_report["mxcheck"] = True
+        if self._tld:
+            args.append("--tld")
+            self.final_report["tld"] = True
+            args.append(self.dictionary_base_path + self._tld_dict)
+
+        args.append(self.observable_name)
+
+        process = subprocess.Popen(
+            args,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        stdout, stderr = process.communicate()
+        logging.warning(stderr.decode("utf-8"))
```

Comment:
Does this work? It would be better if you create an instance of `Logger` outside the function: `logger = logging.getLogger(__name__)` and then call `logger`. You can use as an example the `cymru.py` module.